zndraw/zndraw.py

```diff
@@ -572,12 +572,14 @@ def register_modifier(
         """
         if run_kwargs is None:
             run_kwargs = {}
+        config = GlobalConfig.load()
+        schema = self._update_class_schema(cls, config)
```

Comment:
If we have this here, this will only work for `vis.register_modifier`. Alternativley, we could do it here and catch all modifiers
https://github.com/zincware/ZnDraw/blob/d160a0051a3abffb663e4db7d9adaba1843ed75e/zndraw/zndraw.py#L487